### ISSUE TITLE
Station bounced radios and intercoms signals go through telecomms

### ISSUE THREAD

**Issue by DamianX** [d3athrow/vgstation13#17199]
#### Description of issue
When speaking into a SBR/intercom the signal will attempt to go through telecomms *unless* the signal is blocked by telecomms. This means:

- you can modify signals coming from those devices with NTSL scripts, as long as you don't set their `$content` to a null value or their `$pass` to 0
- if telecommunications is highly damaged (1% integrity, massive slowdown) but functional, said devices will still be affected by the slowdown, reducing their usefulness

I believe this is not intended.

#### Difference between expected and actual behavior / Steps to reproduce 
- Upload this NTSL script: `$content = "honk";`
- Speak into SBR/intercom
- Expect message to go straight to receiving headsets/SBRs/intercoms
- Message is instead passing through telecomms

#### Length of time in which bug has been known to occur
Forever?

#### Issue bingo
- [x] Issue could be reproduced at least once
- [x] Issue could be reproduced by different players
- [x] Issue could be reproduced in multiple rounds
- [x] Issue happened in a recent (less than 7 days ago) round
- [x] [Couldn't find an existing issue about this](https://github.com/d3athrow/vgstation13/issues)

**Comment by SonixApache** [comment]
Wait so you could potentially filter out buttbots / speaker assemblies that are bleating through an intercomm or SBR?

**Comment by ghost** [comment]
Was it not intended to bypass them so that more creative "silencing" than don't pass actually fully stops the ability of people to scream?

From what he's saying sonix, you can't block it without it functioning as it used to, and if you do the other means that "silence" it, it also means it's completely impossible to talk about a reasonably skilled malf/subv silicon or reasonably skilled player that stops by telcomms for a sec.

**Comment by DamianX** [comment]
you can sort of filter them out if you do `$content = " "`, I guess

**Comment by Aurx** [comment]
There are ways to mute SBRs with NTSL.  
You just have to be clever about it.

**Comment by DamianX** [comment]
Le secret

**Comment by Aurx** [comment]
Yes.  Git gud and learn something beyond copypasting ntsl scripts.
